api/Sources/Api/PairQL/Dashboard/Pairs/RequestPublicKeychain.swift

```diff
@@ -0,0 +1,33 @@
+import Dependencies
+import PairQL
+
+struct RequestPublicKeychain: Pair {
+  static let auth: ClientAuth = .admin
+
+  struct Input: PairInput {
+    var searchQuery: String
+    var description: String
+  }
+}
+
+extension RequestPublicKeychain: Resolver {
+  static func resolve(with input: Input, in context: AdminContext) async throws -> Output {
+    @Dependency(\.postmark) var postmark
+    @Dependency(\.logger) var logger
+
+    let html = """
+    <p>From admin <b>\(
+      context.admin.email
+        .raw
```

Comment:
`.raw` is not what we want here, i think that's something funky from the `Rainbow` coloring library we use. we either want `.rawValue` to get the underlying string, or (preferable, i think), just leave it as `context.admin.email` and it will interpolate into a string just fine. we usually just need `.rawValue` when we have to pass one of these tagged `EmailAddress` to some function that explicitly requires a string